ddp-workspace/projects/ddp-sdk/src/lib/models/activity/activityBooleanQuestionBlock.ts

```diff
@@ -6,7 +6,7 @@ import { BooleanRenderMode } from './booleanRenderMode';
 export class ActivityBooleanQuestionBlock extends ActivityQuestionBlock<boolean> {
     public trueContent: string;
     public falseContent: string;
-    public renderMode: BooleanRenderMode;
+    public renderMode?: BooleanRenderMode;
```

Comment:
the `renderMode` will be mandatory soon. 
[PR#1202](https://github.com/broadinstitute/ddp-angular/pull/1202) accidentally was merged before the back-end implementation.